package.json

```diff
@@ -142,7 +143,7 @@
         "sass-loader": "^7.1.0",
         "shelljs": "^0.8.3",
         "style-loader": "^0.22.1",
-        "stylelint": "^9.5.0",
+        "stylelint": "^12.0.0",
```

Comment:
For now, don't change anything in package.json since there packages with breaking change and we need to be careful when update them.